.github/.dockstore.yml

```diff
@@ -161,3 +161,12 @@ workflows:
         - main
       tags:
         - /.*/
+
+  - subclass: WDL
+    name: SingleSample
```

Comment:
```suggestion
    name: SingleSamplePipeline
```
How about this? Our workflow names usually contain a verb

Reply:
Agree, that reads even better!